client/app/dbaas/logs/translations/Messages_cs_CZ.json

```diff
@@ -226,10 +226,10 @@
  "logs_options_manage_role": "Role",
  "logs_options_manage_input": "Kolektory",
  "logs_options_overview_title": "Informace o funkci",
- "logs_options_modal_reactivate_title": "Aktivovat funkci",
- "logs_options_modal_reactivate_description": "Za účelem reaktivace funkce '{{optionType}}' je zapotřebí kontaktovat zákaznickou podporu.",
+ "logs_options_modal_reactivate_title": "Možnost funkci",
```

Comment:
Please do not touch translation files except for `_fr_FR`

Reply:
I rollback theses translations but kept the en_* lines about data gathering tools status since they are invalid since the beginning in english, despite the french initial content.